system-int-test/test/cmr/system_int_test/search/collection_search_format_test.clj

```diff
@@ -715,24 +716,48 @@
                     :concept-type :collection
                     :format-key :dif10
                     :native-id "CMR-5138-No-DataDates-No-Temporal"})
+          related-urls {:RelatedUrls [{:Description "A test related url description."
+                                       :URL "http://example.com/browse-image"
+                                       :Type "GET RELATED VISUALIZATION"
+                                       :URLContentType "VisualizationURL"
+                                       :GetData {:MimeType "image/png"
+                                                 :Format "png"
+                                                 :Size 10.0
+                                                 :Unit "KB"}}]}
+          concept-umm (-> (umm-spec-collection/collection 1 related-urls)
+                          (umm-spec-collection/collection-concept :umm-json)
+                          ingest/ingest-concept)
+
           _ (index/wait-until-indexed)
           opendata-5138-1 (search/find-concepts-opendata :collection {:concept_id (:concept-id concept-5138-1)})
           opendata-5138-2 (search/find-concepts-opendata :collection {:concept_id (:concept-id concept-5138-2)})
           opendata-5138-3 (search/find-concepts-opendata :collection {:concept_id (:concept-id concept-5138-3)})
+          opendata-umm (search/find-concepts-opendata :collection {:concept_id (:concept-id concept-umm)})
           umm-json-5138-3 (search/find-concepts-umm-json :collection {:concept_id (:concept-id concept-5138-3)})
           opendata-coll-5138-1 (first (get-in opendata-5138-1 [:results :dataset]))
           opendata-coll-5138-2 (first (get-in opendata-5138-2 [:results :dataset]))
           opendata-coll-5138-3 (first (get-in opendata-5138-3 [:results :dataset]))
+          opendata-coll-umm (first (get-in opendata-umm [:results :dataset]))
           umm-json-coll-5138-3 (first (get-in umm-json-5138-3 [:results :items]))
           {:keys [references distribution]} opendata-coll-5138-1]
       ;; Sanity checks that collections ingested
+      (is (= 201 (:status concept-umm)))
       (is (= 201 (:status concept-5138-1)))
       (is (= 201 (:status concept-5138-2)))
       (is (= 201 (:status concept-5138-3)))
-      (testing "collection citation fields in opendata response."
+      (testing "Opendata fields in response."
         (are3 [expected-result field-key opendata-test-collection]
           (is (= expected-result (field-key opendata-test-collection)))
 
+          "graphic-preview-file in response"
+          "http://example.com/browse-image" :graphic-preview-file opendata-coll-umm
+
+          "graphic-preview-type in response"
+          "image/png" :graphic-preview-type opendata-coll-umm
+
+          "graphic-preview-description in response"
+          "A test related url description." :graphic-preview-description opendata-coll-umm
+
```

Comment:
I suppose I should have asked this earlier, but right now the only reason type and description and no other fields were selected to be displayed was because I modeled this off of the example in the ticket. There is no reason we couldn't add more fields if desired. Is this a good set of details about the graphic preview or is there any other information from the RelatedUrl you would like added? @eereiter @chris-durbin